### PR TITLE
[CI:DOCS] podman save: clarify formats and transports

### PR DESCRIPTION
**Pull request by vrothberg**
Mention all supports --format values and put them into the context of
supported transports.

Signed-off-by: Valentin Rothberg <rothberg@redhat.com>

### CODE REVIEW

**Comment by vrothberg** [comment]
@containers/podman-maintainers PTAL

File: docs/source/markdown/podman-save.1.md [hunk 1]
```diff
@@ -35,8 +35,9 @@ Write to a file, default is STDOUT
 
 #### **--format**=*format*
 
-Save image to **oci-archive, oci-dir** (directory with oci manifest type), or **docker-dir** (directory with v2s2 manifest type)
+Save image to **docker-archive**, **oci-archive** (see `containers-transports(5)`), **oci-dir** (`dir` transport with oci manifest type), or **docker-dir** (`dir` transport with v2s2 manifest type).
```
Comment (mtrmac) [inline docs/source/markdown/podman-save.1.md:38]: AFAICS `oci-dir` is `c/image/oci/layout`, not `c/image/directory`.

(BTW `AutocompleteImageSaveFormat` seems to be missing one of the options.)

Comment (vrothberg) [reply docs/source/markdown/podman-save.1.md:38]: Oh, I need to track that down.  It could be a regression from libimage.

Comment (vrothberg) [reply docs/source/markdown/podman-save.1.md:38]: OK. This is not a regression but a surprise to me. I'll update accordingly.

File: docs/source/markdown/podman-save.1.md [hunk 2]
```diff
@@ -35,8 +35,9 @@ Write to a file, default is STDOUT
 
 #### **--format**=*format*
 
-Save image to **oci-archive, oci-dir** (directory with oci manifest type), or **docker-dir** (directory with v2s2 manifest type)
+Save image to **docker-archive**, **oci-archive** (see `containers-transports(5)`), **oci-dir** (`transport` transport), or **docker-dir** (`dir` transport with v2s2 manifest type).
```
Comment (mtrmac) [inline docs/source/markdown/podman-save.1.md:38]: > `transport` transport

a typo?

Comment (vrothberg) [reply docs/source/markdown/podman-save.1.md:38]: Definitely. Thanks for catching! Rushing multiple PRs at once is not my strong suit :^)

**Comment by rhatdan** [comment]
LGTM
@containers/podman-maintainers PTAL

**Review by mtrmac — COMMENTED** [review]
LGTM.

**Pull request merged**